web_registry/src/components/PatientDetail/ActivityProgress.tsx

```diff
@@ -16,18 +16,20 @@ export const ActivityProgress: FunctionComponent = observer(() => {
 
     const logMap = new Map<string, IActivityLog>();
 
+    // TODO Activity Refactor: Question for James, why are we sorting here? Also, should this be a helper function in PatientStore?
```

Comment:
What what I see, no reason for it to be sorted.

I agree we could provide such things via the store.  If we did, we'd need good practices that we always _copy_ such an object before modifying it (i.e., if it's a computed property on the store, then it would only be computed once).

I think that's already true when we access the store, this just reminds me again.

Reply:
If there's "no reason to change" here, could just remove the comment?